symfony/plugins/orangehrmTimePlugin/Api/EmployeeTimesheetItemAPI.php

```diff
@@ -50,6 +52,17 @@ class EmployeeTimesheetItemAPI extends Endpoint implements CrudEndpoint
     public const META_PARAMETER_COLUMNS = 'columns';
     public const META_PARAMETER_TIMESHEET = 'timesheet';
     public const META_PARAMETER_EMPLOYEE = 'employee';
+    public const META_PARAMETER_ALLOWED_ACTIONS = 'allowedActions';
+
+    public const MY_TIMESHEET_ACTION_MAP = [
```

Comment:
```suggestion
    public const TIMESHEET_ACTION_MAP = [
```
shall we move this mapping to timesheet service, since this is not only relevant for API, sometimes controllers also may need

Reply:
done